### DISCUSSION TITLE
Documents of Concepts and Source Code may not match for Hantush and HantushWellModel

### DISCUSSION THREAD

**Discussion by DataGeoranger** [pastas/pastas#391] [Q&A] [answered]
Hello!

Thanks again for the excellent package and learning materials. I am following through and identify some potential inconsistencies that I wonder if I get a clarification from the author.

First, I read the explanation about Hantush versue HantushWellModel under the pastas.readthedocs.io under chapter 4. There is an comparison of the implementation of parameters of A, a, and b in Hantush method and HantushWellModel. Please see link and screenshot below.
https://pastas.readthedocs.io/en/dev/concepts/hantush_response.ipynb.html
![image](https://user-images.githubusercontent.com/47303462/167537854-c048535b-c71a-476c-9d7a-12d57e38c537.png)

However, this doesn't match the explanation of the A, a, and b when I run `help(ps.Hantush)` and `help(ps.HantushWellModel)`. In particular, the definition of A and b got flipped. Please see screenshot below in comparison to the IO document. 

![image](https://user-images.githubusercontent.com/47303462/167538073-9d4de6c5-2859-4fd5-8cba-db18029f0df2.png)
![image](https://user-images.githubusercontent.com/47303462/167538102-07f5d35e-7027-4b45-b58c-4754688068ac.png)

May I ask which explanation (the code or the io example) correct? Thank you very much!

Best,
Jack

**Comment by dbrakenhoff** [comment]
Hi Jack,

Thanks for posting this. You're correct that the definitions are not the same everywhere. I submitted a PR #392 that addresses this, which strives to make the definitions consistent everywhere, both in the notebooks and the docstrings. (The code itself was correct). 

After the PR is checked by the rest, we'll merge it into the develop branch. Once that's done I hope that clears up your question!

And for what it's worth, I think the correct definition for `pastas.Hantush` was given in the table in the [response_functions notebook](https://github.com/pastas/pastas/blob/master/concepts/response_functions.ipynb).

Kind regards,

Davíd

**Reply by DataGeoranger** [reply]
Hi David,

Thank you so much for pointing it out to that notebook. It has great content about well impulse function that I was looking to learn. 

Best Regards,
Jack

**Comment by dbrakenhoff** [comment, marked as answer]
The documentation and notebooks have been made consistent in #393!
